### ISSUE TITLE
Using Raw sockets

### ISSUE THREAD

**Issue by thetutlage** [balderdashy/sails#2272]
Here is the deal , ur over robustness has killed me like anything.. a simple socket connection does not seems to be working even after following this https://gist.github.com/mikermcneil/6598661

Inside my userController i have this

```javascript
	destroy: function(req,res){

		var socket = req.socket;
		var io = sails.io;
		io.sockets.emit('users', {thisIs: 'theMessage'});
      }
```

And on client i have this by using socket.io client lib

```javascript
		var socket = io('http://localhost:1337');
		socket.on('users', function(msg){
			console.log('here');
		});
```

**Comment by CWyrtzen** [comment]
@thetutlage did you figure it out?

**Comment by sailsbot** [comment]
Thanks for posting, @thetutlage.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
